hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/request/file/OMRecoverLeaseRequest.java

```diff
@@ -238,7 +236,7 @@ private RecoverLeaseResponse doWork(OzoneManager ozoneManager,
       openKeyInfo.setModificationTime(Time.now());
       // add to cache.
       omMetadataManager.getOpenKeyTable(getBucketLayout()).addCacheEntry(
-          new CacheKey<>(dbOpenFileKey), CacheValue.get(transactionLogIndex, openKeyInfo));
+          dbOpenFileKey, openKeyInfo, transactionLogIndex);
```

Comment:
I see it's usually use CacheKey and CacheValue to wrapper the data in most places to put data in Cache. Why we don't need it here?

Reply:
Okay, it's below one, actually does the same thing.  Is they any benefit use the following one instead of the current one? 

```
default void addCacheEntry(KEY cacheKey, VALUE value, long epoch) 
```

Reply:
Yes. It is the same. It just makes the expression cleaner. Nicholas added it in a previous jira. I have mentioned that in a previously PR (after its merge) as well:

https://github.com/apache/ozone/pull/5847#discussion_r1461295873